File: conditionals/time_of_day.py
#sariah kamore time of day python
import time
current = time.time()
now = time.ctime(current)
local_time = time.localtime(current)
hour = local_time.tm_hour

if hour >=12 and hour <=16:
    print("good happy afternoon")
elif hour > 16 and hour <= 19:
    print("good great evening")
elif hour >19:
    print("good night")
else:
    print("happy sunny morning")

File: conditionals/time_of_day.c
//sariah kamore time of day c
#include <stdio.h>
#include <time.h>
int main(void){
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm.hour;
    if(hour >= 12 && hour <= 16){
        printf("good afternoon");
    }else if(hour > 16 && hour <= 19){
        printf("good evening");
    }else if(hour > 19){
        printf("good night");
    }else{.
        printf("good sunny morning");
    }
    return 0;
}